cmd/cmd.go

```diff
@@ -1,93 +1,104 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
-	"strings"
 	"time"
 
 	"github.com/Clever/batchcli/runner"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/sfn"
 )
 
 func main() {
-	functionCmd := flag.String("cmd", "", "The command to run")
-	localRun := flag.Bool("local", false, "Local mode - auto-assigns random AWS Batch config")
+	activityName := flag.String("name", "", "The activity name to register with AWS Step Functions")
```

Comment:
`stateName` or `appName` is probably the generic name for this.

Reply:
My reason would be that this is the interface for engineers moving mover their containers to use the new CLI. If they use common names like `appName` or `stateName` as opposed to internal naming around step functions then it would help them "know" what each thing means. Also it would mean not requiring understanding concepts about the backend (in this case step-functions) but just the workflow related ideas which they would already need to get familiar with.

Reply:
IMO the fact that we used the same terminology in `gearcmd` that gearman uses in its docs (e.g. `function` is how they name things) makes it clearer what's going on. The abstractions live in workflow-manager, whereas this is a utility function specific to a technology, and could be used independently of workflow manager. Also, the fact that we're going to split this out into a different repo makes me think multiple backends is not really something that we want, and even if we did it seems premature to design for it. All of these seem like reasons against creating new abstractions/terms on top the terms that AWS has laid out.

Reply:
I'm okay with either direction. I think the tradeoff is of clarify of the CLI vs. education and don't think there is a clear winner. 

Just as an example, we will probably use different activity names for each environment, and so we would have to make sure people are using the appropriate combination of `env vars` and `appName` example we would need to do something like 

`sfncli --activityName $_DEPLOY_ENV--my-app-name` which is hard to do in a docker container `CMD` afaik. My sense was that putting that logic in the cli would help us transition existing code without all repositories explicitly calling this out. This would further be complicated with different apps that use the same Docker image. 

I also do realize that this would be putting Clever specific conventions into the CLI. 

Like I said above though I'm not sure what's the right path here right now, and we can figure out where we fall between encoding our conventions and a standard CLI as we test things out.